deltachat-ios/Helper/Utils.swift

```diff
@@ -40,4 +41,16 @@ struct Utils {
         let window = UIApplication.shared.keyWindow
         return window?.safeAreaInsets.bottom ?? 0
     }
+
+    public static func getInviteLink(context: DcContext, chatId: Int) -> String? {
+        // convert `OPENPGP4FPR:FPR#a=ADDR&n=NAME&...` to `https://i.delta.chat/#FPR&a=ADDR&n=NAME&...`
+        if var data = context.getSecurejoinQr(chatId: chatId), let range = data.range(of: "#") {
```

Comment:
I was a bit confused by `data`, it's a `String`, right?

Reply:
it is a string containing "data" (and nothing human readable :)

i named it "qrCodeData" or so first, however, this is also not correct as it morphs to an URL, therefore i just used "data"